rust/tests/flush.rs

```diff
@@ -16,7 +16,9 @@ fn flush_and_cut_chunks() -> Result<()> {
     let messages = mcap::MessageStream::new(&mapped)?;
 
     let mut tmp = tempfile()?;
-    let mut writer = mcap::Writer::new(BufWriter::new(&mut tmp))?;
+    let mut writer = mcap::WriteOptions::new()
+        .chunk_size(None)
+        .create(BufWriter::new(&mut tmp))?;
```

Comment:
Is the goal here just to be explicit? Asking since the default is to not automatically start new chunks.

Reply:
Just being explicit, since we're testing flush here.